src/kernel/arch/x86/paging.zig

```diff
@@ -323,26 +323,26 @@ pub fn init(mem_profile: *const MemProfile, allocator: *std.mem.Allocator) void
     const p_start = std.mem.alignBackward(@ptrToInt(mem_profile.physaddr_start), PAGE_SIZE_4KB);
     const p_end = std.mem.alignForward(@ptrToInt(mem_profile.physaddr_end) + mem_profile.fixed_alloc_size, PAGE_SIZE_4KB);
 
-    var tmp = allocator.alignedAlloc(Directory, @truncate(u29, PAGE_SIZE_4KB), 1) catch panic(@errorReturnTrace(), "Failed to allocate page directory");
+    var tmp = allocator.alignedAlloc(Directory, @truncate(u29, PAGE_SIZE_4KB), 1) catch |e| panic(@errorReturnTrace(), "Failed to allocate page directory: {}", @errorName(e));
     var kernel_directory = @ptrCast(*Directory, tmp.ptr);
     @memset(@ptrCast([*]u8, kernel_directory), 0, @sizeOf(Directory));
 
     // Map in kernel
-    mapDir(kernel_directory, v_start, v_end, p_start, p_end, allocator) catch panic(@errorReturnTrace(), "Failed to map kernel directory");
+    mapDir(kernel_directory, v_start, v_end, p_start, p_end, allocator) catch |e| panic(@errorReturnTrace(), "Failed to map kernel directory: {}", @errorName(e));
     const tty_addr = tty.getVideoBufferAddress();
     // If the previous mappping space didn't cover the tty buffer, do so now
     if (v_start > tty_addr or v_end <= tty_addr) {
         const tty_phys = virtToPhys(tty_addr);
         const tty_buff_size = 32 * 1024;
-        mapDir(kernel_directory, tty_addr, tty_addr + tty_buff_size, tty_phys, tty_phys + tty_buff_size, allocator) catch panic(@errorReturnTrace(), "Failed to map vga buffer in kernel directory");
+        mapDir(kernel_directory, tty_addr, tty_addr + tty_buff_size, tty_phys, tty_phys + tty_buff_size, allocator) catch |e| panic(@errorReturnTrace(), "Failed to map vga buffer in kernel directory: {}", @errorName(e));
```

Comment:
You don't need the `@errorName` as fmt handles it for you. Also (u may reject), so the line isn't long, have this like: 
```Zig
mapDir(kernel_directory, tty_addr, tty_addr + tty_buff_size, tty_phys, tty_phys + tty_buff_size, allocator) catch |e| {
    panic(@errorReturnTrace(), "Failed to map vga buffer in kernel directory: {}\n", e);
};
```
Also for the other error printing.
Also add new lines (\n) to the end of the logging.

Reply:
Oh cool